fpga_interchange/fasm_generators/xc7/xc7.py

```diff
@@ -142,6 +148,25 @@ def get_iologic_prefix(site_name, tile_type):
 
         return "{}_Y{}".format(m.group(1), io_site_idx)
 
+    @staticmethod
+    def format_feature_value(bits):
+        """
+        Formats a FASM feature value assignment according to the given bits
+        (as a string or any iterable yeilding "0" and "1"). The bit range
+        always starts at the LSB.
+        """
+        count = len(bits)
+        value = "".join(bits[::-1])
+
+        if count == 1:
+            bitrange = "[0]"
+        elif count > 1:
+            bitrange = "[{}:0]".format(count - 1)
+        else:
+            assert False, count
+
+        return "{}={}'b{}".format(bitrange, count, value)
+
```

Comment:
This might belong to some common library, as it is rather a FASM utility rather than an xc7 specific one.

Reply:
Moved to `utils.py`